src/components/AbstractSideNav/AbstractSideNav.tsx

```diff
@@ -19,7 +19,7 @@ export interface IAbstractSideNavProps extends HTMLAttributes<HTMLDivElement> {
 
 export const AbstractSideNav = ({ doc }: IAbstractSideNavProps): ReactElement => {
   const router = useRouter();
-  const subPage = last(router.asPath.split('/'));
+  const subPage = last(router.asPath.split('/')).split('?')[0];
```

Comment:
There is a utility for splitting on `?`
```suggestion
  const basePath = useBaseRouterPath();
  const subPage = last(basePath);
```